video/out/opengl/context_drm_egl.c

```diff
@@ -241,17 +236,40 @@ static void acquire_vt(void *data)
     crtc_setup(ctx);
 }
 
+static void drm_egl_start_frame(struct ra_ctx *ctx)
+{
+    struct priv *p = ctx->priv;
+    if (p->kms->atomic_context) {
+        p->kms->atomic_context->request = drmModeAtomicAlloc();
+        p->native_handles[2] = p->kms->atomic_context->request;
+    }
+}
+
 static void drm_egl_swap_buffers(struct ra_ctx *ctx)
 {
     struct priv *p = ctx->priv;
+    struct drm_atomic_context *atomic_ctx = p->kms->atomic_context;
+    int ret;
```

Comment:
I'd add newline after var declarations

Reply:
that one doesn't show up here, but i also changed it.